src/z_generate_json_schema.prog.xml

```diff
@@ -49,6 +49,23 @@
     </item>
    </TPOOL>
    <I18N_TPOOL>
+    <item>
```

Comment:
set serialize only main language in the repo settings?

![image](https://github.com/SAP/abap-file-formats-tools/assets/5888506/fca5e3d7-9042-4842-b9c6-7c8b38d7d955)

Reply:
Ah no, thanks. Changed it